protocol/xmrtaker/event_test.go

```diff
@@ -80,7 +80,7 @@ func TestSwapState_handleEvent_EventETHClaimed(t *testing.T) {
 
 	// test transferBack functionality
 	s.transferBack = true
-	s.Backend.SetBaseXMRDepositAddress(s.Backend.XMRClient().PrimaryAddress())
+	s.Backend.SetXMRDepositAddress(s.Backend.XMRClient().PrimaryAddress(), s.ID())
```

Comment:
@noot Make sure this is correct. This fixed an error where monero was being claimed with a `transferBack` value of `true`, and a deposit address that was nil. In `newTestSwapStateAndNet` above, the swap is initially being setup with `transferBack` as `false` before we flip it to `true` on the line above.

Reply:
this seems fine, did the test start failing or something? was the base XMR deposit address not set?

Reply:
Yeah, it was failing in my PR. It looks like it worked in master, because the claim was pulling a deposit address using an all zero hash, but in my code the ID was not all zeros. I'm putting together a PR that greatly simplifies this deposit address code.

Reply:
Here's the add-on PR to simplify deposit addresses:
https://github.com/AthanorLabs/atomic-swap/pull/328